app/components/notification-bar.js

```diff
@@ -27,6 +27,25 @@ export default Ember.Component.extend({
     }
   }).on("didInsertElement"),
 
+  iOS: function() {
+    var iDevices = [
+      'iPad Simulator',
+      'iPhone Simulator',
+      'iPod Simulator',
+      'iPad',
+      'iPhone',
+      'iPod'
+    ];
+
+    if (!!navigator.platform) {
```

Comment:
https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript
It nothing but forced type conversion to boolean value